### PR TITLE
Added README and License

### PR DESCRIPTION
**Pull request by mayistcool**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+like cars or markers
+*Note: More functionality will be added over time, as functionality is added the README will also be updated!
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Built With
+
+* [Node.js](https://nodejs.org/en/)
+* [Typescript](https://www.typescriptlang.org/)
+
```
Comment (Bendz2012) [inline README.md:20]: add the NativeUI here
```suggestion
[NativeUI](https://github.com/PichotM/FiveM-NativeUI)

```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+## Getting Started
+### Installation
+
+_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
+
+1. Download the .zip in the [Releases](https://github.com/Bendz2012/BendzDevtools/releases/tag/latest)
+2. Put it into your FiveM Ressources Folder
+3. Add to server-config
+	```sh
+	start BendzDevtools
```
Comment (Bendz2012) [inline README.md]: change to ensure, preferred way from FiveM
```suggestion
	ensure BendzDevtools
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+## License
```
Comment (Bendz2012) [inline README.md:60]: update License wrong Path and wrong License

File: README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+Distributed under the MIT License. See `LICENSE.md` for more information.
+[LICENSE.md](https://github.com/Bendz2012/BendzDevtools/blob/readMe/LICENSE.md)
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGMENTS -->
```
Comment (Bendz2012) [inline README.md]: i don't think this section is needed

**Review by Bendz2012 — CHANGES_REQUESTED** [review]
license already exists, you only have added a license template, not the actual license.
you can just delete it.

Look for comments in the README.md

**Comment by mayistcool** [comment]
Test

**Review by Bendz2012 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Bendz2012) [reply README.md:60]: still wrong License

**Review by Bendz2012 — CHANGES_REQUESTED** [review]
https://github.com/Bendz2012/BendzDevtools/pull/3#discussion_r860143590

**Review by Bendz2012 — APPROVED** [review]

**Pull request merged**